ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -306,3 +308,18 @@ Upgrade Suite Setup
     Set Suite Variable    ${IS_SELF_MANAGED}
     Gather Release Attributes From DSC And DSCI
     Set Expected Value For Release Name
+
+Get Operator Starting Version
+    [Documentation]    Returns the starting version of the operator in the upgrade chain
+    ${rc}    ${out}=    Run And Return RC And Output
+    ...    oc get subscription rhods-operator -n redhat-ods-operator -o yaml | yq '.spec.startingCSV' | awk -F. '{print $2"."$3"."$4}'    # robocop: disable
+    Should Be Equal As Integers    ${rc}    0
+    RETURN    ${out}
+
+Is Starting Version Supported
+    [Documentation]    Returns ${TRUE} if the starting version of the upgrade chain is allowed (i.e. >= minimum allowed
+    ...    version), ${FALSE} otherwise.
+    [Arguments]    ${minimum_version}
+    ${starting_ver}=    Get Operator Starting Version
+    ${out}=     Gte    ${starting_ver}    ${minimum_version}
+    RETURN    ${out}
```

Comment:
maybe it will be better to move these Keywords to a generic location like `Common.robot` to be able to use it also in Smoke tests

Reply:
Done